src/simplewallet/simplewallet.cpp

```diff
@@ -101,6 +101,7 @@ enum TransferType {
 
 namespace
 {
+  const auto allowed_priority_strings = {"default", "unimportant", "normal", "elevated", "priority"};
```

Comment:
If this is going to be user visible, GUI people might have an opinion on the names. I'll ping them.